### PR TITLE
Update migration test data

### PR DESCRIPTION
**Pull request by karzak**

### CODE REVIEW

File: migrate/v0_14/migrate.go [hunk 1]
```diff
@@ -32,9 +32,9 @@ import (
 )
 
 var (
-	GenesisTime   = time.Date(2021, 3, 4, 15, 0, 0, 0, time.UTC)
-	RewardEndTime = time.Date(2022, 2, 25, 14, 0, 0, 0, time.UTC)
-	ClaimEndTime  = time.Date(2026, 2, 25, 14, 0, 0, 0, time.UTC)
+	GenesisTime   = time.Date(2021, 3, 24, 15, 0, 0, 0, time.UTC)
```
Comment (denalimarsh) [inline migrate/v0_14/migrate.go:35]: Why was the duration from genesis time to reward time lengthened by a week? Was 51 weeks and now it's 52.

File: migrate/v0_14/migrate_test.go [hunk 2]
```diff
@@ -111,15 +111,15 @@ func getAccount(accounts authexported.GenesisAccounts, address sdk.AccAddress) a
...
+	bz, err := ioutil.ReadFile(filepath.Join("testdata", "kava-6-incentive-state.json"))
 	require.NoError(t, err)
 	var oldIncentiveGenState v0_11incentive.GenesisState
 	cdc := app.MakeCodec()
 	require.NotPanics(t, func() {
 		cdc.MustUnmarshalJSON(bz, &oldIncentiveGenState)
 	})
 
-	bz, err = ioutil.ReadFile(filepath.Join("testdata", "kava-4-harvest-state.json"))
+	bz, err = ioutil.ReadFile(filepath.Join("testdata", "kava-6-harvest-state.json"))
```
Comment (denalimarsh) [inline migrate/v0_14/migrate_test.go:122]:
```suggestion
	bz, err = ioutil.ReadFile(filepath.Join("testdata", "kava-6-hard-state.json"))
```
Module names should line up even in tests

**Review by denalimarsh — APPROVED** [review]
LGTM. Have a couple of questions but nothing blocking.

File: migrate/v0_14/migrate_test.go (hunk 2 above)
Comment (karzak) [reply migrate/v0_14/migrate_test.go:122]: Technically the module name is `harvest` in the kava-6 exported state, it's only `hard` after migration

File: migrate/v0_14/migrate.go (hunk 1 above)
Comment (karzak) [reply migrate/v0_14/migrate.go:35]: Typo in original - reward rates should have been stable for 52 weeks. A minor point anyway, as they will be updated by governance before then.

**Comment by rhuairahrighairidh** [comment]
nit: the help text in `migrate/cmd.go#MigrateGenesisCmd` could be updated to say v0.14.

**Pull request merged**